packages/messages/src/domains/message.js

```diff
@@ -0,0 +1,14 @@
+const { Schema, ObjectId } = require('mongoose');
+
+const connection = require('../drivers/mongodb/connection');
+
+const User = require('./user');
+
+const Message = new Schema({
+  fromUserId: { type: ObjectId, required: true, ref: User },
+  toUserId: { type: ObjectId, required: true, ref: User },
+  message: { type: String, lowercase: true, trim: true },
```

Comment:
nit-pick: What do you think about renaming this to "text"? to use Message.text instead of Message.message
```suggestion
  text: { type: String, lowercase: true, trim: true },
```